### PR TITLE
Add cmd-eikana 1.1.1

### PR DESCRIPTION
**Pull request by holidayworking**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download cmd-eikana` is error-free.
- [x] `brew cask style --fix cmd-eikana` left no offenses.
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install cmd-eikana` worked successfully.
- [x] `brew cask uninstall cmd-eikana` worked successfully.

### CODE REVIEW

File: Casks/cmd-eikana.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'cmd-eikana' do
+  version '1.1.1'
+  sha256 'c61f82e1dfd6093272a1f441361c33da79ea29279d4de749dd31d2533d7ab18d'
+
+  url "https://ei-kana.appspot.com/download/eikana-#{version}.app.zip"
+  appcast 'https://ei-kana.appspot.com',
+          checkpoint: '72ae6ad7ab527ad4713dcca309d1485716a666ff8c2636434a2c17e2102e9092'
+  name 'cmd-eikana'
```
Comment (vitorgalvao) [inline Casks/cmd-eikana.rb]: [The original name also needs to be included](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/name.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/cmd-eikana.rb (hunk 1 above)
Comment (holidayworking) [inline Casks/cmd-eikana.rb]: fixed.

Comment (vitorgalvao) [inline Casks/cmd-eikana.rb]: It **also** needs to be included, **both**. See the linked documentation.

Comment (holidayworking) [inline Casks/cmd-eikana.rb]: Sorry, I added latin alphabet name.

**Pull request merged**